chips/sam4l/src/pm.rs

```diff
@@ -487,22 +500,69 @@ macro_rules! mask_clock {
         let val = (*PM).$field.get() | ($mask);
         (*PM).$field.set(val);
     });
+
+    ($module:ident: $field:ident & $mask:expr) => ({
```

Comment:
Ohh... this totally makes sense, but it took me an embarrassingly long time to switch mental reading of those characters as tokens to match rather than operators